package/gluon-mesh-batman-adv/luasrc/lib/gluon/upgrade/320-gluon-mesh-batman-adv-client-bridge

```diff
@@ -24,13 +25,35 @@ uci:section('network', 'interface', 'client', {
 
 uci:delete('network', 'client_lan')
 
+uci:section('network', 'route', 'br_client_route', {
+	interface = 'client',
+	target = site.prefix4(),
+})
+
+uci:section('network', 'rule', 'local_node_rule', {
+	src = next_node.ip4 .. '/32',
+	lookup = 2,
+})
+uci:set('network', 'local_node_rule', 'in', 'loopback')
```

Comment:
Please merge this *set* into the *section* above (same for `local_node_rule6` below).

I assume this was not done because `in` is a keyword in Lua. The correct syntax for this (and arbitrary Lua values as table keys) is
```lua
{
  ...
  ['in'] = 'loopback',
},
```

Reply:
... and is the `in` setting even necessary at all?